src/vs/base/browser/ui/breadcrumbs/breadcrumbsWidget.ts

```diff
@@ -17,6 +17,7 @@ export abstract class BreadcrumbsItem {
 	dispose(): void { }
 	abstract equals(other: BreadcrumbsItem): boolean;
 	abstract render(container: HTMLElement): void;
+	abstract supportDrag(): boolean;
```

Comment:
An optional property might be better than a method, e.g `supportsDrag?: boolean`